app/services/food_store.py

```
@@ -1,29 +1,363 @@
+
+def _log_missing_food(food_id: str) -> None:
+    """
+    Track missing food and emit periodic summary logs.
+
+    RU: Отслеживает отсутствующие продукты и периодически выводит сводные логи.
+    EN: Tracks missing food and emits periodic summary logs.
+
+    Args:
+        food_id: The food ID that was not found
+    """
+    global _MISSING_FOOD_COUNTER
```

Flake8 (7.3.0)

[error] `global _MISSING_FOOD_COUNTER` is unused: name is never assigned in scope

(F824)